doozerlib/cli/release_gen_assembly.py

```diff
@@ -476,6 +476,12 @@ def _get_advisories_release_jira(self) -> Tuple[Dict[str, int], str]:
             'extras': -1,
             'metadata': -1,
         }
+
+        # For OCP >= 4.12, also microshift advisory placeholder must be created
+        major, minor = self.runtime.get_major_minor_fields()
+        if major > 4 or minor >= 12:  # exclude 3.11, include any 5.y and 4.12+
```

Comment:
It could more readable to have:

```suggestion
        if (major, minor) >= (4, 6):
```

Reply:
This would fail the check for `major = 4, minor = 11`